src/extensions/default/bramble/lib/ConsoleManagerRemote.js

```diff
@@ -16,12 +16,40 @@
      "clear",
      "time",
      "timeEnd"].forEach(function(type) {
-        console[type] = function() {
-            var args = Array.prototype.slice.call(arguments);
+        console[type] = function() {            
+            var values = [];
+            var args = Array.prototype.slice.call(arguments); 
```

Comment:
Ah yeah, IE 11. Instead of the `Array.from(arguments).map` just use `Array.prototype.map.call(arguments, function(arg) { ... });`